docs/conf/sponsorship/prospectus.rst

```diff
@@ -0,0 +1,214 @@
+
+Our audience is made up of:
+
+-  Technical Writers (40%)
+
+-  Developers (22%)
+
+-  Support Staff (15%)
+
+-  Managers (12%)
+
+-  Community Contributors, Enthusiasts & Other Folks (11%)
+
+Why Sponsor
+=============
+
+By supporting a Write the Docs event, your company will gain not only visibility
+and credibility with front-line technical communicators, but also valuable
+insights that will help you get the most out of your own documentation efforts.
+If you're hiring for docs positions, Write the Docs is also an excellent
+opportunity to meet top-notch talent.
+
+Our conferences are run by team of volunteers, and we work hard to keep ticket
+prices affordable for a broad range of attendees. Your sponsorship makes it
+possible for all sorts of documentarians to attend our events, whether they're a
+solo freelancer or they run a docs team for a tech giant. Becoming a sponsor
+demonstrates your commitment to and support of good documentation, and the
+people who build it.
+
+.. raw:: pdf
+
+   PageBreak
+
+Sponsorship Packages
+====================
+
+The following options are suggested sponsorship levels. We are happy to discuss
+adjustments and custom packages.
+
+Each of these packages applies to *one* conference, either USA or Europe. Ask us
+about discounts packages for sponsoring both conferences.
+
+First Draft
+--------------------
+
+The **First Draft** package is aimed at startups, non-profits and open source
+companies:
+
+-  One (1) tickets
+
+-  Small logo & link on the Write the Docs website
+
+-  Name included in press release and event mailings
+
+-  Display 1 promotional (“Swag”) item at the conference (provided by
+       sponsor)
+
+The **First Draft** package costs **US$750**
+
+
+Second Draft
+----------------
+
+The **Second Draft** package for when you are no longer a startup:
+
+-  Two (2) tickets
+
+-  Large logo & link on the Write the Docs website
+
+-  Name included in press release and event mailings
+
+-  Display 1 promotional (“Swag”) items at the conference (provided by
+       sponsor)
+
+The **Second Draft** package costs **US $1500**
+
+Publisher
+---------------------------
+
+The **Publisher** package highlights your company as a force in the industry and
+community:
+
+-  Five (5) tickets
+
+-  Full size logo & link on the Write the Docs website
+
+-  Name included in press release and event mailings
+
+-  Display 1 promotional (“Swag”) items at the conference (provided by
+       sponsor)
+
+The **Publisher** package costs **US $5000**
+
+
+.. raw:: pdf
+
+   PageBreak
+
+Other Sponsorship Opportunities
+===============================
+
+The following à la carte offerings are available to increase visibility
+at the event.
+
+Financial Aid Sponsorships
+--------------------------
+
+Write the Docs is always trying to make our community bigger and more
+inclusive. Talk to us about sponsoring travel and/or registration
+expenses for an attendee who wouldn’t otherwise be able to attend.
+
+Lanyard
```

Comment:
general question - should we make all the "talk to us"/"ask us" text as mailto links? maybe we can even get fancy and open a new email message with a subject line "Sponsoring Write the Docs" or something like that